app/translations/en.json

```diff
@@ -249,7 +249,10 @@
     "view": "View",
     "restore": "Restore",
     "backup": "Backup",
-    "text-readonly": "This is your seed phrase. Please write it down carefully. It is the only way to restore your actual wallet.",
+    "text-readonly": "Before closing this screen, carefully type the seed phrase of the current wallet. It is the only way you will have to restore/access your current wallet.",
```

Comment:
I prefer "write down" to type...   "write down" doesn't specify _how_ to write the thing down.   Whereas "type" suggests that the user uses a keyboard.

Reply:
done!